pkg/operators/process/process.go

```diff
@@ -0,0 +1,649 @@
+		if packetArray != nil {
+			p.dataSource.Release(packetArray)
+		}
+	}()
+
+	// Add each process to the packet array
+	for _, proc := range processes {
+		// Create a new data element for the array
+		packet := packetArray.New()
+
+		// PID field is always emitted (it's required)
+		p.pidField.PutInt32(packet, int32(proc.PID))
+
+		// Emit optional fields if their accessors are not nil
+		if p.ppidField != nil {
+			p.ppidField.PutInt32(packet, int32(proc.PPID))
+		}
+
+		if p.cmdField != nil {
+			p.cmdField.PutString(packet, proc.Command)
+		}
+
+		if p.cpuField != nil {
+			p.cpuField.PutFloat64(packet, proc.CPUUsage)
+		}
+
+		if p.memoryRSSField != nil {
+			p.memoryRSSField.PutUint64(packet, proc.MemoryRSS)
+		}
+
+		if p.memoryVirtualField != nil {
+			p.memoryVirtualField.PutUint64(packet, proc.MemoryVirtual)
+		}
+
+		if p.threadCountField != nil {
+			p.threadCountField.PutInt32(packet, int32(proc.ThreadCount))
+		}
+
+		if p.stateField != nil {
+			p.stateField.PutString(packet, proc.State)
+		}
+
+		if p.usernameField != nil {
+			p.usernameField.PutUint32(packet, proc.Uid)
+		}
+
+		if p.startTimeField != nil {
+			p.startTimeField.PutUint64(packet, proc.StartTime)
+			p.startTimeStrField.PutString(packet, proc.StartTimeStr.Format(time.RFC3339))
+		}
+
+		// Always emit mount namespace ID
+		p.mountNsIDField.PutUint64(packet, proc.MountNsID)
+
+		// Append the packet to the array
+		packetArray.Append(packet)
+	}
+
+	// Emit the packet array with all processes
+	err = p.dataSource.EmitAndRelease(packetArray)
+	if err != nil {
+		return fmt.Errorf("emitting packet array: %w", err)
+	}
+	// Set to nil so it's not released again in the defer function
+	packetArray = nil
+
+	gadgetCtx.Logger().Debugf("Emitted %d process events", len(processes))
+	return nil
+}
+
+// getProcessInfo reads process information from /proc/{pid}
+func (p *processOperatorInstance) getProcessInfo(pid int, timeDelta float64) (ProcessInfo, error) {
+	procInfo := ProcessInfo{
+		PID: pid,
+	}
+
+	// Get mount namespace ID
+	mntNsID, err := containerutils.GetMntNs(pid)
+	if err == nil {
+		procInfo.MountNsID = mntNsID
+	}
+
+	// Check which fields we need to collect
+	needStatus := p.enabledFields[fieldPPID] || p.enabledFields[fieldCommand] ||
+		p.enabledFields[fieldState] || p.enabledFields[fieldUid] ||
+		p.enabledFields[fieldMemoryRSS] || p.enabledFields[fieldMemoryVirtual] ||
+		p.enabledFields[fieldThreadCount]
+
+	needStat := p.enabledFields[fieldCPUUsage] || p.enabledFields[fieldStartTime]
+
+	// Read process status if needed
+	if needStatus {
+		statusFile := fmt.Sprintf("/proc/%d/status", pid)
+		statusData, err := os.ReadFile(statusFile)
+		if err != nil {
+			return procInfo, fmt.Errorf("reading status file: %w", err)
+		}
+
+		// Parse status file
+		scanner := bufio.NewScanner(strings.NewReader(string(statusData)))
+		for scanner.Scan() {
+			line := scanner.Text()
+			parts := strings.SplitN(line, ":", 2)
```

Comment:
```suggestion
			parts := strings.SplitN(line, ":\t", 2)
```

`\t` is always added: see for example:
https://github.com/torvalds/linux/blob/v6.12/fs/proc/array.c#L444-L445
https://github.com/torvalds/linux/blob/v6.12/fs/proc/array.c#L184-L197